src/zjs_ipm.c

```diff
@@ -98,4 +98,15 @@ void zjs_ipm_register_callback(uint32_t msg_id, ipm_callback_t cb)
 #endif
 }
 
+void zjs_ipm_free_callbacks() {
+    #ifdef CONFIG_X86
```

Comment:
I was simply following what the rest of the file uses.  @jimmy-huang should we be using a different flag than CONFIG_X86 for IPM now?  Or does it still make sense?